Content/src/Server/Server.fsproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
What's the difference?

Reply:
So that would be applicable only to `Azure` option?  Should this be a conditional then?

Reply:
I thought this would've been conditional, yes. @theprash thoughts?

Reply:
@theimowski @isaacabraham, I thought it might be better to avoid a conditional branch on the project type just to reduce complexity a little bit if there's little or no downside in using the `.Web` version for all versions of the template. Aside from the web.config file, it also adds some extra entries in `Server.deps.json` which seem to not be very important, but I don't know what they're for.

I'm happy to make it conditional though...?

Reply:
OK I've done it 😀